connector_easypost/models/stock_picking_rate.py

```diff
@@ -105,17 +105,29 @@ def list_rate_currency_id(self, record):
     @only_create
     def service_id(self, record):
         service_obj = self.env['delivery.carrier']
+        partner_obj = self.env['res.partner']
```

Comment:
I added a partner_id field directly to `stock.picking.rate` to support 10.0 since `delivery.carrier` no longer had a `partner_id` (which `stock.picking.rate` mapped to). Since I ended up re-introducing that field into `delivery.carrier`, I need to undo the changes I made here to support that.